index.html

```diff
@@ -63,277 +68,9 @@ <h3 class="connected">
         </li>
       </ul>
     </section>
-
-    <section class="middle" id="portfolio">
-      <ul class="first-item">
-
-        <li class="port-1"><img src="./Graphics/Snapshoot Portfolio.jpg" alt="portfolio-image-1"></li>
-
-        <li>
-          <ul class="description">
-            <h2 class="tonic">Tonic</h2>
-
-            <ul class="feature">
-              <li class="canopy">CANOPY</li>
-              <img src="./Graphics/Counter.svg" alt="dot separator">
-              <li class="backend">Back End Dev</li>
-              <img src="./Graphics/Counter.svg" alt="dot-separator">
-              <li class="year">2015</li>
-            </ul>
-
-          </ul>
-
-          <div class="contents">
-            A daily selection of privately personalized reads;
-            no accounts or sign-ups required.
-          </div>
-
-          <div class="madewith">
-            <div>html</div>
-            <div>css</div>
-            <div>javascript</div>
-          </div>
-
-          <div class="seeproject">
-            <button type="submit">See Project</button>
-          </div>
-        </li>
-
-      </ul>
-
-      <div class="space"></div>
-      <ul class="second-item">
-
-        <li class="port-1"><img src="./Graphics/Snapshoot Portfolio 2.jpg" alt="portfolio-image-2"></li>
-
-        <li>
-          <ul class="description">
-            <h2 class="multi">Multi-Post Stories</h2>
-
-            <ul class="feature">
-              <li class="canopy">CANOPY</li>
-              <img src="./Graphics/Counter.svg" alt="dot-separator">
-              <li class="backend">Back End Dev</li>
-              <img src="./Graphics/Counter.svg" alt="dot-separator">
-              <li class="year">2015</li>
-            </ul>
-
-          </ul>
-
-          <div class="contents">
-            A daily selection of privately personalized reads;
-            no accounts or sign-ups required.
```

Comment:
- [ ] :thinking: Kindly make sure to stick to the Figma design by using the right background color for your work cards:

Your design:
![Screenshot from 2023-02-01 18-12-19](https://user-images.githubusercontent.com/97834160/216104032-857500bb-5d03-4c5e-af44-b57cced0ca35.png)

Figma:
![Screenshot from 2023-02-01 18-14-34](https://user-images.githubusercontent.com/97834160/216104405-b15c5c69-0df2-4411-8697-d0e6a2909d4f.png)